docs/pages/deployment/backup-restore.rst

```diff
@@ -64,28 +64,11 @@ To restore a backup, follow the following steps:
 
 1. shutdown the node
 2. remove the following directories from the ``datadir``: ``events``, ``network``, ``vcr``, and ``vdr``
-3. keep node offline by removing ``network.bootstrapnodes`` and ``network.grpcaddr`` from the config file and set ``network.enablediscovery`` to ``false``
-4. follow the restore procedure for your storage (BBolt, Redis, Hashicorp Vault)
-5. restore the ``vcr/trusted_issuers.yaml`` file inside ``datadir``
-6. start your node
-7. make empty POST calls to
-
-.. code-block:: http
-
-    POST <node-address>/internal/network/v1/reprocess?type=application/vc+json
-    POST <node-address>/internal/network/v1/reprocess?type=application/ld+json;type=revocation
-
-.. note::
-
-    When making the API calls, make sure you use the proper URL escaping (``%2B`` for ``+`` and ``%3B`` for ``;``).
-    Reprocess calls return immediately and will do the work in the background.
-
-8. wait until the reprocess operations are complete
-9. shutdown the node
-10. undo the configuration changes in step 3
-11. start the node
+3. follow the restore procedure for your storage (BBolt, Redis, Hashicorp Vault)
```

Comment:
Reprocess isn't required at all anymore, since we restore leia stores from backup Bbolt db?

nice